src/components/frame/v5/pages/UserAdvancedPage/partials/UserAdvancedSettings/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface AdvancedSettingsFields {
```

Comment:
please generete this interface using AdvancedSettingsFields form src/components/frame/v5/pages/UserAdvancedPage/types.ts file and Omit.